android/src/main/jni/rnscreens.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <ReactCommon/JavaTurboModule.h>
+#include <ReactCommon/TurboModule.h>
+#include <jsi/jsi.h>
+#include <react/renderer/components/rnscreens/RNSScreenComponentDescriptor.h>
+
+namespace facebook {
+namespace react {
+
+JSI_EXPORT
```

Comment:
Is `JSI_EXPORT` used here in some kind of official guide? If not, why not `RNSCREENS_EXPORT`?

Reply:
I went with usage of `JSI_EXPORT` because it handles broader set of compilers (not only GNU). I could provide custom macro implementation, but:

1. JSI's one is ready to be used
2. If we wanted to achieve similar functionality to `JSI_EXPORT` it would require to detect what compiler is being used - I guess it's good idea to use already estabilished solutions.